books/ion-1-1/src/binary/e_expressions.md

```diff
@@ -0,0 +1,427 @@
+
+## Encoding Expressions
+
+> [!NOTE]
+> This chapter focuses on the binary encoding of e-expressions. [_Macros by example_](../macros_by_example.md) explains what they are and how they are used.
+
+### E-expression with the address in the opcode
+
+If the value of the opcode is less than `64` (`0x40`), it represents an E-expression invoking the macro at the
+corresponding __address__—-an offset within the local macro table.
+
+#### Invocation of macro address `7`
+```
+┌──── Opcode in 00-3F range indicates an e-expression
+│     where the opcode value is the macro address
+│
+07
+└── FixedUInt 7
+```
+
+#### Invocation of macro address `31`
+```
+┌──── Opcode in 00-3F range indicates an e-expression
+│     where the opcode value is the macro address
+│
+1F
+└── FixedUInt 31
+```
+
+Note that the opcode alone tells us which macro is being invoked, but it does not supply enough information for the
+reader to parse any arguments that may follow. The parsing of arguments is described in detail in the section _Macro
+calling conventions_. (TODO: Link)
+
+
+#### E-expression With the Address as a Trailing `FixedUInt`
+
+While E-expressions invoking macro addresses in the range `[0, 63]` can be encoded in a single byte using
+<<e_expression_with_the_address_in_the_opcode, E-expressions with the address in the opcode>>,
```

Comment:
This doesn't render as a link. This syntax occurs in several other locations in this file too.

Reply:
This was leftover Asciidoc syntax, thanks.